backend/templates/emails/ownershiprequest.html

```diff
@@ -0,0 +1,17 @@
+<!-- TYPES:
```

Comment:
nit: `ownership_request.html`, and rename `request.html` to `membership_request.html`